metastore/iceberg-metastore/src/main/java/org/apache/drill/metastore/iceberg/MetastoreContext.java

```diff
@@ -50,4 +51,11 @@
    * @return transformer instance
    */
   Transformer<T> transformer();
+
+  /**
+   * Returns expiration handler that expires Iceberg table metadata.
+   *
+   * @return expiration handler instance
+   */
+  ExpirationHandler expirationHandler();
```

Comment:
This ```MetastoreContext``` interface consists of getters without get prefix... Looks a little bit strange for me. But I'm ok with it if it does not violate the Drill's conventions.